net-mgmt/snmp/pkg-descr

```diff
@@ -0,0 +1,6 @@
+ bsnmp. It allows you to use SNMPv1 and SNMPv2 to query your OPNsense appliance
```

Comment:
no, they are helpful in identifying the daemon is in itself not overly helpful :)